src/WebJobs.Extensions.DocumentDB/Config/DocumentDBConfiguration.cs

```diff
@@ -38,7 +45,86 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException("context");
             }
 
-            context.Config.RegisterBindingExtension(new DocumentDBAttributeBindingProvider(context.Config, this, context.Trace));
+            INameResolver nameResolver = context.Config.GetService<INameResolver>();
+            IConverterManager converterManager = context.Config.GetService<IConverterManager>();
+
+            BindingFactory factory = new BindingFactory(nameResolver, converterManager);
+
+            IBindingProvider outputProvider = factory.BindToGenericAsyncCollector<DocumentDBAttribute, DocumentDBContext>(typeof(DocumentDBAsyncCollector<>),
```

Comment:
New rule is in the SDK, so this can be updated. 

---
In reply to: [63896758](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/97#discussion_r63896758) [](ancestors = 63896758,63895687)